phpBB/styles/prosilver/template/viewtopic_body.html

```diff
@@ -237,63 +255,35 @@ <h3 {% if postrow.S_FIRST_ROW %}class="first"{% endif %}>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_REPORT -->
-						<li>
-							<a href="{postrow.U_REPORT}" title="{L_REPORT_POST}" class="button button-icon-only">
-								<i class="icon fa-exclamation fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_REPORT}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_WARN -->
-						<li>
-							<a href="{postrow.U_WARN}" title="{L_WARN_USER}" class="button button-icon-only">
-								<i class="icon fa-exclamation-triangle fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_WARN}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_INFO -->
-						<li>
-							<a href="{postrow.U_INFO}" title="{L_INFORMATION}" class="button button-icon-only">
-								<i class="icon fa-info fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_INFORMATION}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
-					<!-- IF postrow.U_QUOTE -->
-						<li>
-							<a href="{postrow.U_QUOTE}" title="{L_REPLY_WITH_QUOTE}" class="button button-icon-only">
-								<i class="icon fa-quote-left fa-fw" aria-hidden="true"></i><span class="sr-only">{L_BUTTON_QUOTE}</span>
-							</a>
-						</li>
-					<!-- ENDIF -->
+					<!-- INCLUDE posting_buttons.html -->
 					<!-- EVENT viewtopic_body_post_buttons_after -->
+						<li class="post-buttons-responsive-menu dropdown-container">
+							<a href="javascript:void(0);" class="button button-icon-only post-button-responsive dropdown-toggle">
+								{{ Icon('iconify', 'fa:bars', '', true, 'c-button-icon') }}
+							</a>
+							<div class="dropdown">
+								<div class="pointer">
+									<div class="pointer-inner"></div>
+								</div>
+							<ul class="dropdown-contents">
+								<!-- EVENT viewtopic_body_post_buttons_before -->
+								<!-- INCLUDE posting_buttons.html -->
+								<!-- EVENT viewtopic_body_post_buttons_after -->
+							</ul>
+						</div>
+					</li>
 				</ul>
 			<!-- ENDIF -->
 		<!-- ENDIF -->
 		<!-- EVENT viewtopic_body_post_buttons_list_after -->
 
 			<!-- EVENT viewtopic_body_postrow_post_details_before -->
 			<p class="author">
-				<!-- IF S_IS_BOT -->
-					<span><i class="icon fa-file fa-fw <!-- IF postrow.S_UNREAD_POST -->icon-red<!-- ELSE -->icon-lightgray<!-- ENDIF --> icon-md" aria-hidden="true"></i><span class="sr-only">{postrow.MINI_POST}</span></span>
-				<!-- ELSE -->
-					<a class="unread" href="{postrow.U_MINI_POST}" title="{postrow.MINI_POST}">
-						<i class="icon fa-file fa-fw <!-- IF postrow.S_UNREAD_POST -->icon-red<!-- ELSE -->icon-lightgray<!-- ENDIF --> icon-md" aria-hidden="true"></i><span class="sr-only">{postrow.MINI_POST}</span>
-					</a>
-				<!-- ENDIF -->
+				{% if postrow.S_UNREAD_POST %}
+					<span title="{postrow.MINI_POST}">{{ Icon('iconify', 'fa:file', postrow.MINI_POST, true, 'c-topic-unread-icon') }}</span>
```

Comment:
What's the benefit of this additional span wrap? As the title is also already added to the icon.

Reply:
not sure why i added the title there but the span was already there as a. wrapper so we should keep it for now.